packages/devtools_app/lib/src/timeline/flutter/event_details.dart

```diff
@@ -110,6 +110,11 @@ class EventSummary extends StatelessWidget {
   Widget build(BuildContext context) {
     return ListView(
       children: [
+        ListTile(
```

Comment:
It looks like tests cover the existence of these tiles.  Please update.

Reply:
done.